src/AdminConsole/Pages/Organization/CreateApplication.cshtml.cs

```diff
@@ -166,14 +167,38 @@ private async Task InitializeAsync()
 
     public class CreateApplicationForm
     {
-        [Required, MaxLength(60)]
-        public string Name { get; set; }
+        private string _name;
 
-        [Required, MaxLength(62)]
+        [Required, MaxLength(60), MinLength(3)]
+        public string Name
+        {
+            get
+            {
+                return _name;
+            }
+            set
+            {
+                _name = HtmlSanitizer.Sanitize(value);
```

Comment:
Shouldn't we send `Bad Request` instead of silently changing user input?

Reply:
Because it is a page handler that either returns the same page or redirects to a new one. Returning a bad request would trigger the exception handler and redirect to the error page.

What you probably mean is you want to validate the form input, compare sanitized with unsanitized input and invalidate the form.

Reply:
Yes, sorry. Forgot it was in AC. Essentially I'm wondering if communicating an error is better than silently fixing it.